packages/terra-list/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Restored the `refCallback` prop for the ListSectionHeader component to be applied to the intended `li` element instead of the inner `div` tag. 
```

Comment:
```suggestion
  * Restored the `refCallback` prop for the ListSectionHeader and ListSubSectionHeader component to be applied to the intended `li` element instead of the inner `div` tag. 
```